go.mod

```diff
@@ -173,6 +206,8 @@ require (
 replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	// celestia-core
+	github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v0.38.11-0.20250331172655-bbd3b9bdd1ee
```

Comment:
Seems like we should create a celestia-core release so this can use an official release. Is everything merged in celestia-core and you just need a release?

Reply:
If you want to cut a rc/alpha tag on core then that would be great. We could pick that up here. 
I'm not 100% sure if everything is merged. There was still some discussion around mempool, right?

Reply:
I don't have enough context on the v0.38.x branch and what the plan is regarding celestia-core releases moving forward so I don't feel comfortable cutting a release from that branch. 

@tac0turtle what's the status of v0.38.x in celestia-core? Is it safe to cut release candidates from?